tests/in/python-dephell_changelogs.spec

```diff
@@ -0,0 +1,70 @@
+#
```

Comment:
Do you really need the whole specfile here?
You can just create file with artifical %prep lines to test it...